salt/modules/opkg.py

```diff
@@ -456,6 +466,10 @@ def remove(name=None, pkgs=None, **kwargs):  # pylint: disable=unused-argument
     if not targets:
         return {}
     cmd = ['opkg', 'remove']
+    if kwargs.get('remove_dependent_packages', False):
+        cmd.append('--force-removal-of-dependent-packages')
+    if kwargs.get('auto_remove', False):
```

Comment:
You actually not need to write default `False` to `.get(...)`. In this case perfectly enough to:
```python
if kwargs.get('auto_remove'):
    cmd.append...
```
It will either return `None` (no param) or any value. You can validate it though, if you want to _force_ people pass `True/False` instead of `1/0` etc.